text/0000-safe-mem-forget.md

```diff
@@ -0,0 +1,124 @@
+- Feature Name: N/A
+- Start Date: 2015-04-15
+- RFC PR: (leave this empty)
+- Rust Issue: (leave this empty)
+
+# Summary
+
+Alter the signature of the `std::mem::forget` function to remove `unsafe`.
+Explicitly state that it is not considered unsafe behavior to not run
+destructors.
```

Comment:
I (and others on IRC) think the priority is backwards here. The RFC should first and foremost be about declaring leaks safe, or more generally (in the Alternative section) addressing the ability to leak resources/prevent `drop` from being called. Making `mem::forget` safe is just a corollary, not the core issue.